.circleci/unittest/windows/scripts/run_test.sh

```diff
@@ -6,5 +6,5 @@ eval "$(./conda/Scripts/conda.exe 'shell.bash' 'hook')"
 conda activate ./env
 
 python -m torch.utils.collect_env
-pytest --cov=torchaudio --junitxml=test-results/junit.xml -v --durations 20 test
-flake8 torchaudio test
```

Comment:
this is not tested separately, hence why this is removed here, right?

Reply:
that's correct.